tests/blessed_images/generated_blessed_images.py

```diff
@@ -37,6 +37,6 @@ def main(blessed_images_dir, tests_data_dir):
     img.complete()
 
 if __name__ == '__main__':
-    blessed_images_dir = os.path.dirname(os.path.realpath(__file__))
+    blessed_images_dir = os.path.abspath(os.path.join(os.path.dirname(os.path.realpath(__file__)), "../../../worldengine-data/tests/images"))
```

Comment:
I know I'm a bit late but I just noticed this:
Works for me, but seems a little too hard-coded to me. Is there maybe a better way that can be patched in?